### PR TITLE
docs(metrics): Document dgraph_txn_aborts_total metric

### PR DESCRIPTION
**Pull request by aaroncarey**
**Still blocked by CLA Assistant, no matter what I change on my client or how many times I try to sign the CLA**

Document the new dgraph_txn_aborts_total metric that counts transaction aborts for a Dgraph server cluster.

Also:
* Added an introduction sentence
* Shifted headers to sentence-case,
* Minor rewording for usage and style 

Fixes DGRAPH-2606

---
This change is [Reviewable](https://reviewable.io/reviews/dgraph-io/dgraph/6927)

 
 
 
 
 
 

Docs Preview: [Dgraph Preview](https://dgraph-298412bd30-110406.surge.sh)

### CODE REVIEW

**Review by danielmai — CHANGES_REQUESTED** [review]
> Reviewed 1 of 1 files at r2.
> *[Reviewable](https://reviewable.io/reviews/dgraph-io/dgraph/6927)* status: all files reviewed, 1 unresolved discussion (waiting on @aaroncarey, @darkn3rd, @MichaelJCompton, and @OmarAyo)

___
*[wiki/content/deploy/metrics.md, line 66 at r2](https://reviewable.io/reviews/dgraph-io/dgraph/6927#-MMZPBV75UEJWwicgIDZ:-MMZPBV842yliaMwknTl:b-4chn7p) ([raw file](https://github.com/dgraph-io/dgraph/blob/b59d4884a756420a2acae3213c542cd99d5b36fa/wiki/content/deploy/metrics.md#L66)):*
> ```
> on the cluster.
> ```
Metrics are mostly local to that one Alpha. So, it's not right to say "cluster" here.

If users want to know the total number of aborts across the cluster, they can sum up this metric across all Alphas.

**Review by danielmai — APPROVED** [review]
One comment and then :lgtm: 

> *[Reviewable](https://reviewable.io/reviews/dgraph-io/dgraph/6927#-:-MMZPcv6-2FZsStoJt5x:b-q4nvk6)* status: all files reviewed, 1 unresolved discussion (waiting on @aaroncarey, @darkn3rd, @MichaelJCompton, and @OmarAyo)

File: wiki/content/deploy/metrics.md [hunk 1]
```diff
@@ -7,12 +7,13 @@ weight = 15
 +++
 
 
-Dgraph metrics follow the [metric and label conventions for
-Prometheus](https://prometheus.io/docs/practices/naming/).
+Dgraph database provides metrics on disk activity, memory usage, Dgraph instance activity
+and server node health - along with built-in metrics provided by Go. Dgraph metrics follow
```
Comment (OmarAyo) [inline wiki/content/deploy/metrics.md:11]: I would avoid using the "dash" inside a sentence, I suggest replacing it with a coma or semicolon.

`.. and server node health, along with built-in metrics...`

**Review by OmarAyo — APPROVED** [review]
> Reviewed 1 of 1 files at r3.
> *[Reviewable](https://reviewable.io/reviews/dgraph-io/dgraph/6927)* status: all files reviewed, 2 unresolved discussions (waiting on @aaroncarey, @darkn3rd, and @MichaelJCompton)

**Review by aaroncarey — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/dgraph-io/dgraph/6927)* status: 0 of 1 files reviewed, 2 unresolved discussions (waiting on @danielmai, @darkn3rd, @MichaelJCompton, and @OmarAyo)

___
*[wiki/content/deploy/metrics.md, line 66 at r2](https://reviewable.io/reviews/dgraph-io/dgraph/6927#-MMZPBV75UEJWwicgIDZ:-MMZrjqj8fLNjwv3umww:b-ncjvsr) ([raw file](https://github.com/dgraph-io/dgraph/blob/b59d4884a756420a2acae3213c542cd99d5b36fa/wiki/content/deploy/metrics.md#L66)):*
<details><summary><i>Previously, danielmai (Daniel Mai) wrote…</i></summary><blockquote>

> ```
> on the cluster.
> ```
Metrics are mostly local to that one Alpha. So, it's not right to say "cluster" here.

If users want to know the total number of aborts across the cluster, they can sum up this metric across all Alphas.
</blockquote></details>

Fixed, thanks for the clarification :)
___
*[wiki/content/deploy/metrics.md, line 11 at r3](https://reviewable.io/reviews/dgraph-io/dgraph/6927#-MMZMmjTN5uiB5SDU4pE-r3-11:-MMaBjAHCt7eX4CWKIXO:bb6un1j) ([raw file](https://github.com/dgraph-io/dgraph/blob/9d10aaa676d542f24f0acb258606e47478d1c1c3/wiki/content/deploy/metrics.md#L11)):*
<details><summary><i>Previously, OmarAyo (OmarAyo) wrote…</i></summary><blockquote>

I would avoid using the "dash" inside a sentence, I suggest replacing it with a comma or semicolon.

`.. and server node health, along with built-in metrics...`
</blockquote></details>

Done

**Review by aaroncarey — COMMENTED** [review]
:lgtm:

> Reviewed 1 of 1 files at r3.
> *[Reviewable](https://reviewable.io/reviews/dgraph-io/dgraph/6927#-:-MMaLryL-3VEyspnI29g:bnfp4nl)* status: 0 of 1 files reviewed, 2 unresolved discussions (waiting on @aaroncarey, @danielmai, @darkn3rd, @MichaelJCompton, and @OmarAyo)

**Review by aaroncarey — COMMENTED** [review]
> Dismissed @OmarAyo from a discussion.
> *[Reviewable](https://reviewable.io/reviews/dgraph-io/dgraph/6927)* status: 0 of 1 files reviewed, 1 unresolved discussion (waiting on @aaroncarey, @danielmai, @darkn3rd, @MichaelJCompton, and @OmarAyo)

**Review by aaroncarey — COMMENTED** [review]
> Dismissed @danielmai from a discussion.
> *[Reviewable](https://reviewable.io/reviews/dgraph-io/dgraph/6927)* status: 0 of 1 files reviewed, all discussions resolved (waiting on @aaroncarey, @darkn3rd, @MichaelJCompton, and @OmarAyo)

**Review by aaroncarey — COMMENTED** [review]
> Reviewed 1 of 1 files at r4.
> *[Reviewable](https://reviewable.io/reviews/dgraph-io/dgraph/6927)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @darkn3rd and @MichaelJCompton)

**Review by darkn3rd — APPROVED** [review]
> *[Reviewable](https://reviewable.io/reviews/dgraph-io/dgraph/6927)* status: 0 of 1 files reviewed, 1 unresolved discussion (waiting on @aaroncarey and @MichaelJCompton)

*[a discussion](https://reviewable.io/reviews/dgraph-io/dgraph/6927#-MMbV6qYC3jtZWi2GUzT:-MMbV6qYC3jtZWi2GUzU:bg7igkv) (no related file):*
:lgtm: 

---

**Review by aaroncarey — COMMENTED** [review]
> Reviewed 1 of 1 files at r6.
> Dismissed @darkn3rd from a discussion.
> *[Reviewable](https://reviewable.io/reviews/dgraph-io/dgraph/6927)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @MichaelJCompton)

**Comment by aaroncarey** [comment]
I have fixed my local client, but this PR is still blocked by CLA Assistant.

**Review by aaroncarey — COMMENTED** [review]
> Reviewed 1 of 1 files at r7.
> *[Reviewable](https://reviewable.io/reviews/dgraph-io/dgraph/6927)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @MichaelJCompton)

**Pull request merged**